common.php

```diff
@@ -13281,13 +12859,13 @@ class_exists( 'EWWW\ExactDN' ) &&
 <div class='ewww-settings-notices'>
 	<?php ewwwio()->settings_errors(); ?>
 	<?php if ( ewww_image_optimizer_get_option( 'ewww_image_optimizer_webp_enabled' ) ) : ?>
-		<?php ewww_image_optimizer_notice_webp_bulk(); ?>
+		<?php ewwwio()->notices->webp_bulk_notice(); ?>
 	<?php endif; ?>
 	<?php if ( ! empty( $_GET['ewww_pngout'] ) ) : ?>
-		<?php ewww_image_optimizer_pngout_installed(); ?>
+		<?php ewwwio()->notices->pngout_installed_notice(); ?>
 	<?php endif; ?>
 	<?php if ( ! empty( $_GET['ewww_svgcleaner'] ) ) : ?>
-		<?php ewww_image_optimizer_svgcleaner_installed(); ?>
+		<?php ewwwio->notices->svgcleaner_installed_notice(); ?>
```

Comment:
Found the source of the PHP 7.4 error, missing parentheses for the ewwwio() call.